lib/src/layer/tile_layer/tile.dart

```diff
@@ -1,125 +1,74 @@
 import 'package:flutter/widgets.dart';
-import 'package:flutter_map/flutter_map.dart';
+import 'package:flutter_map/src/core/point.dart';
+import 'package:flutter_map/src/layer/tile_layer/tile_builder.dart';
+import 'package:flutter_map/src/layer/tile_layer/tile_image.dart';
+
+class Tile extends StatefulWidget {
```

Comment:
The diff is confusing because of renaming. Tile was renamed to TileImage and AnimatedTile to Tile.